.github/workflows/tests-l4lb-v1.10.yaml

```diff
@@ -185,7 +197,7 @@ jobs:
           target_url: ${{ env.check_url }}
 
       - name: Set commit status to failure
-        if: ${{ failure() }}
+        if: ${{ !success() }}
```

Comment:
The `cancelled` event is handled separately for Slack notifications (see code below), so unlike the above we should not switch from `failure` to `!success` here.